napari/layers/points/points.py

```diff
@@ -672,6 +692,38 @@ def current_size(self, size: Union[None, float]) -> None:
             self.refresh()
             self.events.size()
 
+    @property
+    def fixed_size(self):
+        """bool: maintain point size fixed regardless of zoom"""
+        return self._fixed_size
+
+    @fixed_size.setter
+    def fixed_size(self, value) -> bool:
+        self._fixed_size = bool(value)
+        self.events.fixed_size()
+
+    @property
+    def antialias(self):
+        """float: amount in pixels of antialiasing"""
+        return self._antialias
+
+    @antialias.setter
+    def antialias(self, value) -> Union[int, float]:
+        if value < 0:
+            value = 0
+        self._antialias = value
+        self.events.antialias()
+
+    @property
+    def spherical(self):
+        """bool: display spherical shading and lighting on points"""
+        return self._spherical
```

Comment:
Small doubt about naming because I would never think to set the `spherical` property of a non-circle symbol but that's totally possible with what you've done! 

We could solve this in a similar way to what was proposed above with `fixed_size`. Maybe `Points.appearance` which can be `'flat'` or `'3d'`.  It could also be fun to automatically switch this on ndisplay change, flat for 2d and 3d for 3d...

Reply:
Switching automatically would be fun, but maybe not "there" yet (shading is fixed with no control over the lights, for example, which may cause confusion/odd looking scenes). I also think a bool is better here, as well. I would have gone with `3d` already in vispy, if only python allowed me to :P

Reply:
Okay let's wait on autoswitch - do you agree that spherical is confusing? What about `Points.shading`? This could feasibly be a bool although I would still prefer an enum/strings

Reply:
Yes, spherical makes you think about spheres only. `shading` would have the mesh counterpart, which is nice, but it also does not suggest that points will appear "3d"...

Reply:
`shading='flat'/'3d'` 🙃

Reply:
About autoswitch: the relationship between 2D disc and 3D sphere is the only "obvious" one. All the other markers will appear to always be facing one direction, which is not exactly obvious. On the other hand, it might be a good way to quickly know "ok, we are in 3D now!".

Reply:
I prefer `shading` too - I think we have it as an enum in the surface layer. It might be a nice idea to make this part of the API match that one if we can https://github.com/napari/napari/blob/5f96d5d814aad697c367bdadbb1a57750e2114ad/napari/layers/surface/_surface_constants.py#L7-L26

Reply:
What do you mean by matching the API -- using the same Enum? If yes, which one of the existing ones would work with this application?

Reply:
hmm maybe it doesn't quite work perfectly. Maybe we just want `shading` as the bool and to be `True`/ `False`? Does @kevinyamauchi have thoughts here